### DISCUSSION TITLE
Command palette's override of ctrl-K is disruptive

### DISCUSSION THREAD

**Discussion by HebaruSan** [community/community#15196] [Other Feature Feedback, Questions, & Ideas] [answered]
<kbd>Ctrl-K</kbd> is my habitual browser hotkey to start a search. The command palette overrides the default browser behavior with its own functionality. I have disabled the command palette to be able to continue initiating a search from a tab with a GitHub page.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @HebaruSan  👋🏼  Thanks for the feedback, we do have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.

**Reply by HebaruSan** [reply]
Thanks, that works for me!
